proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/EidasSamlParserResponse.java

```diff
@@ -21,14 +21,18 @@
     @ValidDestinationUriString
     private String assertionConsumerServiceLocation;
 
+    @JsonProperty
+    private boolean transientPid;
```

Comment:
I'd prefer to have this boolean named `eidasTransientPid` like the others (or vice-versa), plus name for the getter.

I'm a bit torn on whether the `eidas` prefix is redundant - look forward to other reviewer's thoughts

Reply:
@CharlesIC any thoughts on the naming?